test/pkg/environment/aws/expectations.go

```diff
@@ -214,3 +214,17 @@ func (env *Environment) GetCustomAMI(amiPath string, versionOffset int) string {
 	Expect(err).To(BeNil())
 	return *parameter.Parameter.Value
 }
+
+func (env *Environment) ExpectRunInstances(instanceInput *ec2.RunInstancesInput) *ec2.Reservation {
+	// implement IMDSv2
```

Comment:
Include `GinkgoHelper()` at the top so that we properly get the right line if one of the expectations fails.